recipes/Ragna/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "Ragna" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Ragna-{{ version }}.tar.gz
+  sha256: c4da922760b2ab55cb5b6c9bbe010768ad0f7e1d1f7e0f07fbf9ce6ce42ee84b
+
+build:
+  skip: true  # [py<39]
+  entry_points:
+    - ragna = ragna.__main__:app
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=45
+    - setuptools-scm >=6.2
+    - pip
+  run:
+    - python >=3.9
+    - aiofiles
+    - emoji
+    - fastapi
+    - httpx
+    - importlib-metadata >=4.6  # [py<310]
```

It's OK to have this empty dependency for newer Pythons
```suggestion
    - importlib-metadata >=4.6
```